internal/commands/result.go

```diff
@@ -729,25 +771,54 @@ func printTableRow(title string, counts *wrappers.EngineResultSummary, statusNum
 		fmt.Printf(formatString, title, counts.High, counts.Medium, counts.Low, counts.Info, scanFailedString)
 	case scanCanceledNumber:
 		fmt.Printf(formatString, title, counts.High, counts.Medium, counts.Low, counts.Info, scanCanceledString)
+	case scanPartialNumber:
+		fmt.Printf(formatString, title, counts.High, counts.Medium, counts.Low, counts.Info, scanPartialString)
 	default:
 		fmt.Printf(formatString, title, counts.High, counts.Medium, counts.Low, counts.Info, scanSuccessString)
 	}
 }
 
+func printSCSSummary(microEngineOverviews []*wrappers.MicroEngineOverview) {
+	fmt.Printf("              Supply Chain Security Results\n")
+	fmt.Printf("              ---------------------------------------------------------------     \n")
+	fmt.Println("              |                      High   Medium   Low   Info   Status    |")
+	for _, microEngineOverview := range microEngineOverviews {
+		printSCSTableRow(microEngineOverview)
+	}
+	fmt.Printf("              ---------------------------------------------------------------     \n\n")
+}
+
+func printSCSTableRow(microEngineOverview *wrappers.MicroEngineOverview) {
+	formatString := "              | %-16s   %4d   %6d   %4d   %4d   %-9s  |\n"
+	notAvailableFormatString := "              | %-16s   %4s   %6s   %4s   %4s   %5s      |\n"
+
+	riskSummary := microEngineOverview.RiskSummary
+	microEngineName := microEngineOverview.FullName
+
+	switch microEngineOverview.Status {
+	case scsScanUnavailableString:
+		fmt.Printf(notAvailableFormatString, microEngineName, notAvailableString, notAvailableString, notAvailableString, notAvailableString, notAvailableString)
+	default:
+		fmt.Printf(formatString, microEngineName, riskSummary[highLabel], riskSummary[mediumLabel], riskSummary[lowLabel], riskSummary[infoLabel], microEngineOverview.Status)
+	}
+}
+
 func printResultsSummaryTable(summary *wrappers.ResultSummary) {
 	totalHighIssues := summary.EnginesResult.GetHighIssues()
 	totalMediumIssues := summary.EnginesResult.GetMediumIssues()
 	totalLowIssues := summary.EnginesResult.GetLowIssues()
 	totalInfoIssues := summary.EnginesResult.GetInfoIssues()
+	totalIssues := summary.TotalIssues + summary.ScsIssues
```

Comment:
Because we don't support the Markdown and HTML reports yet. If we were counted in summary.TotalIssues we would have our results in the total in those reports but be absent in the rest of those reports.